worker/index.js

```diff
@@ -18,5 +18,7 @@ const env = conf.environment[conf.nodeEnv];
 
 const dbSample = require('../db/index').Sample;
 
-setInterval(() => dbSample.doTimeout(), env.checkTimeoutIntervalMillis);
+if (conf.enableWorkerDynos === true || conf.enableWorkerDynos === 'true') {
```

Comment:
in general, I think we should put all of these 'true'/true conditions for env vars in config.js so that the modules which use them can just do a simple `if (conf.enableWorkerDynos)...`